libcloud/container/drivers/kubernetes.py

```diff
@@ -30,20 +33,178 @@
 from libcloud.container.providers import Provider
 from libcloud.container.types import ContainerState
 
-__all__ = ["KubernetesContainerDriver"]
+from libcloud.compute.types import NodeState
+from libcloud.compute.base import Node
+from libcloud.compute.base import NodeSize
+from libcloud.compute.base import NodeImage
+from libcloud.common.exceptions import BaseHTTPError
+
+__all__ = [
+    "KubernetesContainerDriver",
+    "to_n_bytes",
+    "to_memory_str",
+    "to_cpu_str",
+    "to_n_cpus",
+]
 
 
 ROOT_URL = "/api/"
 
 
+K8S_UNIT_MAP = OrderedDict(
+    {
+        "K": 1000,
+        "Ki": 1024,
+        "M": 1000 * 1000,
+        "Mi": 1024 * 1024,
+        "G": 1000 * 1000 * 1000,
+        "Gi": 1024 * 1024 * 1024,
+    }
+)
+
+
+def to_n_bytes(memory_str: str) -> int:
```

Comment:
It also wouldn't hurt to add direct test cases for all those utility conversion functions :)

Reply:
Sorry for the delay, I added the test cases